SU-HSDO/suhumsci
# (feat)STN-105: vertical card news

# READY FOR REVIEW

## Summary
[JIRA - STN-105](https://sparkbox.atlassian.net/browse/STN-105)

This pull request modifies the vertical card to mimic the horizontal card. The styles were refactored so that both horizontal and vertical cards will use the same base styles while using modifier classes to account for different layouts. This will also allow us to use the card classes for other card types.

## Steps to Test
1. The setup for the vertical card should be very similar to the horizontal card. I can pair with whoever reviews this card to get that setup locally.
2. Create a new basic page with 4 views: 
    -  Vertical card (display: list)
    -  Vertical card (display: featured)
    -  Horizontal card (display: list)
    -  Horizontal card (display: featured)
3. Ensure that the refactored styles do not affect the horizontal card and it still has the correct breakpoint styles.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
